packages/node_modules/@node-red/editor-client/src/sass/base.scss

```diff
@@ -15,6 +15,9 @@
 **/
 
 
+body {
```

Comment:
We try not to set global styles for things like this as it could cause issues when trying to embed the editor into another page. However setting `overflow: hidden` on `.red-ui-editor` would not make sense as it would clip the tooltip unexpectedly.

The alternative is to set this as an inline style on `index.mst`.

I'll merge this as-is for now - but may be something we change later.